src/components/nutritional_Information/food/FoodCategory.test.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { render, screen, waitFor } from "@testing-library/react";
+import { Route } from "react-router";
+import userEvent from "@testing-library/user-event";
+import FoodCategory from "./FoodCategory";
+import RoutingTestWrapper from "../../../tests/RoutingTestWrapper";
+import FoodCategoryData from "../../../data/nutritional_information/FoodCategoryData";
+
+describe("Food Category Component", () => {
+  test("renders with no category", async () => {
+    const route = "/NutritionalInformation";
+
+    const { asFragment } = render(
+      <RoutingTestWrapper path={route}>
+        <Route path={route} element={<FoodCategory />} />
+      </RoutingTestWrapper>
+    );
+
+    expect(screen.getByTestId("menutitle-title")).toBeVisible();
```

Comment:
If explore food families text is the same bit of html as the element with data-testid="menu-title" then you can just do the text one.